### PR TITLE
update translations

### PR DESCRIPTION
**Pull request by Simon-Laux**
and add `română` to language selection (18.05.2023)

### CODE REVIEW

File: _locales/_languages.json [hunk 1]
```diff
@@ -29,6 +29,7 @@
   "pl": "Polski",
   "pt": "Português",
   "pt_BR": "Português do Brasil",
+  "ro": "română",
```
Comment (r10s) [inline _locales/_languages.json:32]:
```suggestion
  "ro": "Română",
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -7,6 +7,8 @@
 - update deltachat-node and deltachat/jsonrpc-client to `v1.115.0`
 - remove message id from chatlistitem
 - Improved message search results
+- Update translations (18.05.2023)
+- add `română` language
```
Comment (r10s) [inline CHANGELOG.md:11]:
```suggestion
- add Romanian translation
```

(the changelog is english, in the UI, however, we show the local name)

**Review by r10s — APPROVED** [review]

File: _locales/_languages.json (hunk 1 above)
Comment (Simon-Laux) [reply _locales/_languages.json:32]: not sure how Romanian does capitalisation, but I guess you did the research

Comment (r10s) [reply _locales/_languages.json]: assuming that proper nouns and language names at the beginning of a phrase are written in upper case is a good and reasonable default. exceptions can be handled then :)

but yes, i am also looking is used on comparable lists - and if questions pop up, i ask the translator who is doing the translation.

**Pull request merged**